src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -41,6 +41,88 @@ public static FileInfo GetGuidNameForNewFile(string test, string extension)
         }
 
         [Fact]
+        public void T201_FileFormatException()
+        {
+            var e = new FileFormatException();
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T202_FileFormatException()
+        {
+            var e2 = new IOException("Test");
+            var e = new FileFormatException("Test", e2);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T203_FileFormatException()
+        {
+            var partUri = new Uri("/idontexist.xml", UriKind.Relative);
+            var e = new FileFormatException(partUri);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T203A_FileFormatException()
+        {
+            Uri partUri = null;
+            var e = new FileFormatException(partUri);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T204_FileFormatException()
+        {
+            var partUri = new Uri("/idontexist.xml", UriKind.Relative);
+            var e = new FileFormatException(partUri, "Test");
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T205_FileFormatException()
+        {
+            var partUri = new Uri("/idontexist.xml", UriKind.Relative);
+            var e2 = new IOException("Test");
+            var e = new FileFormatException(partUri, e2);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T205A_FileFormatException()
+        {
+            Uri partUri = null;
+            var e2 = new IOException("Test");
+            var e = new FileFormatException(partUri, e2);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T206_FileFormatException()
+        {
+            var partUri = new Uri("/idontexist.xml", UriKind.Relative);
+            var e2 = new IOException("Test");
+            var e = new FileFormatException(partUri, "Test", e2);
+            Assert.NotNull(e);
+        }
+
+        [Fact]
+        public void T208_InvalidParameter()
+        {
+            var ba = File.ReadAllBytes("plain.docx");
+            var documentPath = "document.xml";
+            Uri partUriDocument = PackUriHelper.CreatePartUri(new Uri(documentPath, UriKind.Relative));
+
+            using (MemoryStream ms = new MemoryStream())
+            {
+                ms.Write(ba, 0, ba.Length);
```

Comment:
The stream's position is going to be after all the written bytes... is that what you intended?  I'd have expected this to either do ```ms.Position = 0;``` after the write, or instead of the write just construct the MemoryStream with the bytes, e.g. ```new MemoryStream(ba)```.